UnityProject/Assets/Scripts/Core/Chat/ChatRelay.cs

```diff
@@ -252,13 +255,15 @@ private void CheckForRadios(ChatEvent chatEvent)
 			{
 				foreach (var slots in playerScript.DynamicItemStorage.ServerContents.Values)
 				{
+					if (SBRSpamCheck == true) break;
 					foreach (var slot in slots)
 					{
 						if (slot.IsEmpty) continue;
 						if (slot.Item.TryGetComponent<LocalRadioListener>(out var listener)
 						    && listener != chatEvent.originator)
 						{
 							listener.SendData(chatEvent);
+							SBRSpamCheck = true;
```

Comment:
whereas this Reset?

Reply:
It doesnt need to be.

The whole spam prevention is that it only finds one nearby radio and thats it, don't bother searching for more